src/tools/RelayInternalTypes.js

```diff
@@ -23,7 +23,7 @@ import type RelayQuery from 'RelayQuery';
 
 export type Call = {
   name: string,
-  type?: string,
+  type?: ?string,
```

Comment:
Yeah, this already is marked as an optional field.

@papigers I apologize for the back and forth on this. Since this is already an optional field, how about reverting the addition of all the `type: null`s to focus this PR on just the logic & results that actually change?